web-app-deploy/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM microsoft/azure-cli:2.0.47
 
 LABEL version="1.0.0"
 LABEL maintainer="Microsoft Corporation"
-LABEL com.github.actions.name="GitHub Action for Azure Web Apps"
+LABEL com.github.actions.name="Deploy to Azure WebApp"
```

Comment:
Should there be space between web and app?

Reply:
added. thanks